Sources/ReactorKit/Pulse.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  Pulse.swift
+//  ReactorKit
+//
+//  Created by tokijh on 2021/01/11.
+//
+
+@propertyWrapper
+public struct Pulse<Value> {
+
+  public var value: Value {
+    didSet {
+      self.riseValueUpdatedCount()
+    }
+  }
+  var valueUpdatedCount = UInt.min
+
+  public init(wrappedValue: Value) {
+    self.value = wrappedValue
+  }
+
+  public var wrappedValue: Value {
+    get { return self.value }
+    set { self.value = newValue }
+  }
+
+  public var projectedValue: Pulse<Value> {
+    return self
+  }
+
+  private mutating func riseValueUpdatedCount() {
+    if self.valueUpdatedCount == UInt.max {
+      self.valueUpdatedCount = UInt.min
+    } else {
+      self.valueUpdatedCount += 1
+    }
+  }
+}
+
+extension Pulse: Equatable {
+  public static func == (lhs: Pulse<Value>, rhs: Pulse<Value>) -> Bool {
+    return lhs.valueUpdatedCount == rhs.valueUpdatedCount
```

Comment:
```suggestion
    return lhs.valueUpdatedCount == rhs.valueUpdatedCount 
      && lhs.value == rhs.value
```

How about comparing the value together?
I think there will be a problem when comparing pulses.

Reply:
Good idea! I fixed it :) @OhKanghoon